server/bin/state/config/pbench-server.cfg

```diff
@@ -13,11 +13,8 @@ debug_unittest = True
 commit_id = unit-test
 environment = unit-test
 admin-email = unit-test-user@example.com
-
-[Indexing]
-server = elasticsearch.example.com:9280
-index_prefix = pbench-unittests
-bulk_action_count = 2000
+put-token = 71ac1fe3f0743aa710e82000e553ec30
```

Comment:
Do we want to commit this to a public Git repo?

Reply:
It is already public and it is a fake token.